first_ruby/Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
```

Comment:
Whats this file for?

Reply:
This file is where the app's gem dependencies are declared. Gems are simply plugins, packages or library.

Reply:
How is this file different than `Gemfile.lock`?

Reply:
`Gemfile` takes dependency like from specific range, while `Gemfile.lock` records the specific version the dependency project is used.